css/src/atomics/typography.scss

```diff
@@ -0,0 +1,112 @@
+// Heading sizes
+.heading-h1 {
+	font-size: $font-size-1 !important;
+}
+
+.heading-h2 {
+	font-size: $font-size-2 !important;
+}
+
+.heading-h3 {
+	font-size: $font-size-3 !important;
+}
+
+.heading-h4 {
+	font-size: $font-size-4 !important;
+}
+
+.heading-h5 {
+	font-size: $font-size-5 !important;
+}
+
+.heading-h6 {
+	font-size: $font-size-6 !important;
+}
+
+// Non-heading sizes
```

Comment:
Super huge nitpick: space before and after comments?

Reply:
👍